docker/php/Dockerfile

```diff
@@ -5,6 +5,8 @@ RUN apt-get update && apt-get install -y \
     && docker-php-ext-install \
         iconv mcrypt mbstring intl pdo pdo_mysql pdo_pgsql
 
+RUN echo "date.timezone=Europe/Paris" > /usr/local/etc/php/conf.d/date_timezone.ini
```

Comment:
In the image phpfpm, it's the path used for ini files.
As it is linked to this image, I don't see any problem for that.

On Fri, Feb 26, 2016 at 10:02 AM, 0wnrepo <notifications@github.com> wrote:

> In docker/php/Dockerfile
> <https://github.com/wallabag/wallabag/pull/1727#discussion_r54218406>:
>
> > @@ -5,6 +5,8 @@ RUN apt-get update && apt-get install -y \
> >      && docker-php-ext-install \
> >          iconv mcrypt mbstring intl pdo pdo_mysql pdo_pgsql
> >
> > +RUN echo "date.timezone=Europe/Paris" > /usr/local/etc/php/conf.d/date_timezone.ini
>
> isn't this a hardcoded path ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wallabag/wallabag/pull/1727/files#r54218406>.
>